Source/BusinessEntities/BusinessEntities/BusinessEntities.csproj

```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Move this csproj under the Adapters solution folder

Reply:
Should the folder structure Adapters > Business Entities > User 
or something like
Adapters > UserBE